src/styles/utils/header.scss

```diff
@@ -0,0 +1,97 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+/* logo settings rools */
+
+.logo {
+  align-self: center;
+  width: $logo-proportion;
+  height: $logo-proportion;
+
+  &-img {
+    display: block;
+  }
+}
+
+/* end of logo settings rools */
+
+/* start of menu  et navigations links rools */
+
+.menu {
+  &-nav {
+    list-style: none;
+    display: flex;
+    justify-content: end;
+    margin: 0;
+    padding: 0;
+  }
+
+  &-li {
+    text-decoration: none;
+    position: relative;
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    /* #navigation  section and active link
+
+I wrote this section in this way because npm tests don't passe if I put two classes
+at the same time in the attibute <a> (.menu-link and .is-active)
```

Comment:
strange, but thanks for explanation - I see you know it's not the best way to add styles 👍